util/src/dns.rs

```diff
@@ -219,25 +219,18 @@ pub async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     // enable logging early
     let log_level = if opts.debug {
-        log::LevelFilter::Debug
+        Some(Level::DEBUG)
     } else if opts.info {
-        log::LevelFilter::Info
+        Some(Level::INFO)
     } else if opts.warn {
-        log::LevelFilter::Warn
+        Some(Level::WARN)
     } else if opts.error {
-        log::LevelFilter::Error
+        Some(Level::ERROR)
     } else {
-        log::LevelFilter::Off
+        None
     };
```

Comment:
this could use `tracing`'s [`LevelFilter` type](https://docs.rs/tracing/latest/tracing/level_filters/struct.LevelFilter.html), which has an `OFF` variant.

Reply:
Let me look at cleaning that up, I think there was some reason that I went in this direction instead, but that might have changed in subsequent changes.

Reply:
I think the logic I have now is a little closer to what I want, i.e. `Option` vs `Off` would be "default" when `None` rather than `Off` which would disable those events entirely.